extras/encryption/README.md

```diff
@@ -1,21 +1,19 @@
-Moor version that uses the
-[sqflite_sqlcipher](https://github.com/davidmartos96/sqflite_sqlcipher) instead of
-sqflite.
+Drift version that uses the [sqflite_sqlcipher](https://github.com/davidmartos96/sqflite_sqlcipher) instead of sqflite.
```

Comment:
I see it's already broken in the existing version, but:

```suggestion
Drift version that uses the [sqflite_sqlcipher](https://github.com/davidmartos96/sqflite_sqlcipher) package instead of sqflite.
```

Reply:
Apologies for the delay, somehow missed the notification. Updated.